ui/window.go

```diff
@@ -198,19 +213,18 @@ func (win *Window) HandleEvents() {
 		case <-win.IsShutdown:
 			// backend processes shutdown is complete, exit UI process too.
 			return
-		case e := <-win.Events():
+		case e := <-events:
```

Comment:
Move this implementation to default switch case where you will use a loop to recursively check for new events from `win.Event()` function until `DestroyEvent` is detected.

Reply:
why we need do that? please check this example on gioui: https://github.com/gioui/gio-example/blob/4aa16fb00aeeffc801f08c049eddc4087f69b322/kitchen/kitchen.go#L136